cmd/hiffy/src/lib.rs

```diff
@@ -401,13 +403,19 @@ fn hiffy(context: &mut humility::ExecutionContext) -> Result<()> {
     let subargs = HiffyArgs::try_parse_from(subargs)?;
 
     if subargs.list {
-        hiffy_list(hubris, subargs.verbose)?;
+        hiffy_list(hubris, subargs.filter)?;
         return Ok(());
+    } else if !subargs.filter.is_empty() {
+        bail!("filters can only be provided with --list");
```

Comment:
Nit: I don't think we can get here; `filter` already has the clap attribute `requires = "list"` so it should bail for us if we get filters without `--list`.

Reply:
I will remove the `requires = "list"` which doesn't, it turns out, actually work in this context. :|